assets/js/util.js

```diff
@@ -49,6 +49,29 @@
 	$("#orderNeededDate").attr("min", dtToday);
 	});
 
+	/**
+	 *  Pre-fill order form based on user selection from the homepage
+	 */
+	$(".button").click(function() {
+		localStorage.setItem("form-data", $(this).data("order-type"));
+	});
+
+	$(document).ready(() => {
+		var data = localStorage.getItem("form-data");
+		$("#event-type").val(data);
+		
+		// select cake or cupcakes on order form based on user selection
+		if (data === "cupcakes") {
+			$(".cakeOrCupcakes option[value='cupcakes']").prop("selected", true).change();
```

Comment:
add a TODO comment here stating that we will flesh this out in the future after we get more info from our client.

From the client we will need to know cake size, cake flavor, frosting flavor, etc, in order to fill out the form properly